spec/lib/submission_builder/state_return_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'rails_helper'
 
 describe SubmissionBuilder::StateReturn do
-  StateFile::StateInformationService.active_state_codes.each do |state_code|
-    describe '.build', required_schema: state_code do
+  # NC does not include W2s in the state return xml
+  StateFile::StateInformationService.active_state_codes.excluding("nc").each do |state_code|
```

Comment:
No change necessary, totally subjective - you could avoid the comment and maybe improve readability by saying `states_requiring_w2s = StateFile::StateInformationService.active_state_codes.excluding("nc")` and then iterating over that list